Automattic/jetpack
# Search: add date selector to extended search layout

## Proposed changes:

* On multisite, search results in "expanded" include format include the Post Date.
* A Team51 partner would like the post date implemented in their search results, though they are not on multisite.
* This PR introduced a new checkbox in Customberg to allow the post date to be turned on/off in the expanded search result format.
* Multisite enabled sites will have this checkbox defaulted to enabled, to ensure no immediate formatting change.
* Standard sites will have this checkbox defaulted to disabled, for the same reasons.

### Other information:


## Jetpack product discussion



Could not find any existing discussion regarding this. As noted this a request from a Team51 partner, though I feel this would add value to the wider community.

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Testing a standard site: After applying patch, ensure post dates **are not** shown in  search results.
* Head to the instant search options panel inside Jetpack, enable post dates in the "extended" format.
* Ensure post dates now show on search results.


* Testing a multisite: After applying patch, ensure post dates **are** shown in  search results.
* Head to the instant search options panel inside Jetpack, ensure that the post date setting operates as expected. 


![Screenshot 2023-02-13 at 15 45 15](https://user-images.githubusercontent.com/3149464/218504469-eb88e33d-c3d2-44f1-81fa-27e531ba09af.png)
